README.md

```diff
@@ -1,2 +1,2 @@
-# Genesis
+# Pegesus
 A new programming language with syntax similarities to C++ and Python
```

Comment:
Lets make similarities to Bash, Golang, and Rust as well and deploy to Kubernetes over Apache Cloud